web/src/js/components/ProxyApp.jsx

```diff
@@ -37,6 +38,10 @@ class ProxyAppMain extends Component {
         this.onSettingsChange = this.onSettingsChange.bind(this)
     }
 
+    componentWillMount() {
+        this.props.wsConnect()
```

Comment:
I'm not entirely convinced - what's the benefit of moving the WebSocket connection into React?